File: Easy_python/760_find_anagram_mappings.py
class Solution:
    #hashmap
    def anagramMappings(self, A: List[int], B: List[int]) -> List[int]:
        hashmap={}
        returnarray=[]
        for idx,number in enumerate(B):
            hashmap[number]=idx
        for idx,number in enumerate(A):
            returnarray.append(hashmap[A[idx]])
        return returnarray

File: Easy/760_Find_anagram_mappings.cpp
class Solution {
public:
    vector<int> anagramMappings(vector<int>& A, vector<int>& B) {
        map<int,int> hashmapB;
        for(int i=0;i < B.size();i++)
        {
            hashmapB[B[i]] = i;
        }
        vector<int> res;
        for(int i=0;i < A.size();i++)
        {
            res.push_back(hashmapB[A[i]]);
        }
        return res;
    }
};
